model/src/main/java/com/basistech/rosette/dm/Dependency.java

```diff
@@ -0,0 +1,128 @@
+/*
+* Copyright 2016 Basis Technology Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package com.basistech.rosette.dm;
+
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * An inter-token dependency from a parser.
+ * Note that a token index of {@code -1} means 'ROOT'.
+ */
+public class Dependency extends BaseAttribute {
+    private final String relationship;
+    private final int governorTokenIndex;
```

Comment:
``Mention`` contains only offsets into the raw text, not token indexes.  This is so that
the Mention would be valid even if tokenization changed.  Does the same argument hold
for ``Dependency`` or is this fundamentally token-based?